ovos_plugin_manager/utils/audio.py

```
@@ -0,0 +1,640 @@
+                flac_converter,
+                "--stdout",
+                "--totally-silent",
+                # put the resulting FLAC file in stdout, and make sure it's not mixed with any program output
+                "--best",  # highest level of compression available
+                "-",  # the input FLAC file contents will be given in stdin
+            ],
+            stdin=subprocess.PIPE,
+            stdout=subprocess.PIPE,
+            startupinfo=startup_info,
+        )
+        flac_data, stderr = process.communicate(wav_data)
```

Ruff (0.14.10)

Unpacked variable `stderr` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)